lib/matplotlib/pyplot.py

```diff
@@ -154,6 +155,13 @@ def post_execute():
 
             _IP_REGISTERED = post_execute
             _INSTALL_FIG_OBSERVER = False
+
+            # trigger IPython's eventloop integration, if available
+            from IPython.core.pylabtools import backend2gui
+
+            ipython_gui_name = backend2gui.get(get_backend())
```

Comment:
:sheep: right.  Thought that `backend2gui.get` was something much more complicated.